symphony-bdk-core/src/main/java/com/symphony/bdk/core/activity/model/ActivityInfo.java

```diff
@@ -27,4 +27,7 @@
   /** Description of the activity (can contain multiple lines) */
   private String description;
 
+  /** Whether the bot mention is required or not */
+  private boolean requiresBotMention;
```

Comment:
to me this looks like this is specific to a command activity only and should not belong here

Reply:
As discussed, we could have an arbitrary key uniquely identifying the activity, which could be a string, a list or an object whose actual type would depend on the activity.